[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:72 Selenium Manager binary found at: C:\Users\sharath.ls\OneDrive - Feuji Software Solutions Pvt Ltd\Documents\Python\PytestSeleniumlogZip\PytestSelenium\.venv\Lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:129 Executing process: C:\Users\sharath.ls\OneDrive - Feuji Software Solutions Pvt Ltd\Documents\Python\PytestSeleniumlogZip\PytestSelenium\.venv\Lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --debug --output json
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 chromedriver not found in PATH
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 chrome detected at C:\Program Files\Google\Chrome\Application\chrome.exe
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 Running command: wmic datafile where name='C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe' get Version /value
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 Output: "\r\r\n\r\r\nVersion=122.0.6261.129\r\r\n\r\r\n\r\r\n\r"
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 Detected browser: chrome 122.0.6261.129
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 Required driver: chromedriver 122.0.6261.128
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 chromedriver 122.0.6261.128 already in the cache
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 Driver path: C:\Users\sharath.ls\.cache\selenium\chromedriver\win64\122.0.6261.128\chromedriver.exe
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:146 Browser path: C:\Program Files\Google\Chrome\Application\chrome.exe
[35mDEBUG   [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:107 Using driver at: C:\Users\sharath.ls\.cache\selenium\chromedriver\win64\122.0.6261.128\chromedriver.exe
[35mDEBUG   [0m selenium.webdriver.common.service:service.py:219 Started executable: `C:\Users\sharath.ls\.cache\selenium\chromedriver\win64\122.0.6261.128\chromedriver.exe` in a child process with pid: 13228 using 0 to output -3
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54265/session {"capabilities": {"firstMatch": [{}], "alwaysMatch": {"browserName": "chrome", "pageLoadStrategy": "normal", "browserVersion": null, "goog:chromeOptions": {"extensions": [], "binary": "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe", "args": []}}}}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:244 Starting new HTTP connection (1): localhost:54265
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:546 http://localhost:54265 "POST /session HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":{"capabilities":{"acceptInsecureCerts":false,"browserName":"chrome","browserVersion":"122.0.6261.129","chrome":{"chromedriverVersion":"122.0.6261.128 (f18a44fedeb29764b2b5336c120fdd90ef1a3f5c-refs/branch-heads/6261@{#1057})","userDataDir":"C:\\Users\\sharath.ls\\AppData\\Local\\Temp\\scoped_dir13228_664724307"},"fedcm:accounts":true,"goog:chromeOptions":{"debuggerAddress":"localhost:54286"},"networkConnectionEnabled":false,"pageLoadStrategy":"normal","platformName":"windows","proxy":{},"setWindowRect":true,"strictFileInteractability":false,"timeouts":{"implicit":0,"pageLoad":300000,"script":30000},"unhandledPromptBehavior":"dismiss and notify","webauthn:extension:credBlob":true,"webauthn:extension:largeBlob":true,"webauthn:extension:minPinLength":true,"webauthn:extension:prf":true,"webauthn:virtualAuthenticators":true},"sessionId":"a3ff2e100b2e2950f505c561ccd3c5dd"}} | headers=HTTPHeaderDict({'Content-Length': '888', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54265/session/a3ff2e100b2e2950f505c561ccd3c5dd/window/maximize {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:546 http://localhost:54265 "POST /session/a3ff2e100b2e2950f505c561ccd3c5dd/window/maximize HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":{"height":688,"width":1296,"x":-8,"y":-8}} | headers=HTTPHeaderDict({'Content-Length': '51', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54265/session/a3ff2e100b2e2950f505c561ccd3c5dd/url {"url": "https://www.saucedemo.com/"}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:546 http://localhost:54265 "POST /session/a3ff2e100b2e2950f505c561ccd3c5dd/url HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
[32mINFO    [0m root:test_login.py:16 Starting test_invalid_signin
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:313 DELETE http://localhost:54265/session/a3ff2e100b2e2950f505c561ccd3c5dd {}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:546 http://localhost:54265 "DELETE /session/a3ff2e100b2e2950f505c561ccd3c5dd HTTP/1.1" 200 0
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
[35mDEBUG   [0m selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request